bin/run_web.sh

```diff
@@ -2,28 +2,29 @@
 
 # This Source Code Form is subject to the terms of the Mozilla Public
 # License, v. 2.0. If a copy of the MPL was not distributed with this
-# file, You can obtain one at https:://mozilla.org/MPL/2.0/.
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
+# Runs the webapp.
 
-# For development, it probably makes sense to use one process (``--workers=1``)
-# that can handle multiple concurrent connections (``--worker-connections=4``).
-# The number of connections you want to handle simultaneously depends on your
-# setup and all that.
-#
-# Make sure you use the ``gevent`` worker class (``--worker-class=gevent``).
-# Otherwise it's not going to use the gevent WSGI app and then you're not going
-# to be able to handle multiple network connections concurrently.
+set -euxo pipefail
```

Comment:
We want this to be `set -euxo pipefail` so it has tracing on and tells us the values of the environment variables.

Reply:
Also, fun fact that since we had scripts in two places, we had to `run_web.sh` scripts. Oops.